examples/axisartist/demo_parasite_axes2.py

```diff
@@ -1,45 +1,52 @@
+"""
+Parasite Axis Demo
+
+The following code is an example of of a parasite axis.
+It aims to show a user how to plot multiple different values
+onto one single plot. Notice how in this example, par1 and
+par2 are both calling twinx meaning both are tied directly to
+the x-axis. From there, each of those two axis can behave 
+seperately from the current x-axis as well as the other
```

Comment:
I don't quite understand. The previous sentence says that both are tied to the x-axis, but this sentence says they can behave separately.